### DISCUSSION TITLE
Choose ViewColumn (editor group) when selecting from quick access menu (Command Pallete)

### DISCUSSION THREAD

**Discussion by Salads** [microsoft/vscode-discussions#1736] [Extension Development QnA] [answered]
I'm particular about having just two side by side views but also being able to choose which editor group the file/symbol location is opened in. Presumably I'd have a hotkey via a new command...

I see there's functions to open a file with a specified view column via `window.showTextDocument`, 
but I want to use the existing command pallete / quick access menu for opening a file(Ctrl + P) / goto symbol (Ctrl + T). 

Is there a way to get the quick access menu's currently selected item?

**Comment by robconery** [comment]
Hi @Salads - is this a question about using VS Code or creating an extension? This discussion forum is dedicated to the latter (see #5). If it's a question about extension development, would you mind adding a bit more detail?

**Reply by Salads** [reply]
Hi, sorry about the confusion

I'm trying to make an extension. (I assume this can't be done in VSCode without changes)

Say I have two editor groups, and I only want 2 editor groups, side by side. In VSCode, there is a command to "open to side", using the command pallate, for reference I mean this popup that appears when I press "Ctrl + P" (Open File) or "Ctrl + T" (Goto symbol in workspace) for example.
![image](https://github.com/user-attachments/assets/df2bb29e-ad04-411d-a3d9-925dac71036b)

With that window, if I select something and press "Ctrl  + Enter", it only opens "besides" the currently active editor, so I have to specifically select the first group before doing so as a workaround. 

And so I'd like to create an extension that lets me pick exactly which editor group the file will open in. I just don't know how to get which item in that command palette is currently selected.

**Reply by robconery** [reply]
Thanks for the detail! @isidorn is this something you can help with?

**Reply by isidorn** [reply, marked as answer]
Not possible from an extension. For this to work we would have to open up the quick pick menu to extensions, and to pass the selected item as context. We do not do that right now. But it is a fair feature request that we open this up.

**Reply by Salads** [reply]
In that case I'd like to have this be a feature request, where would I put that?

**Reply by robconery** [reply]
You can find more information on how to submit a feature request here - and thanks so much for doing that!
https://github.com/microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions. Closing for now - feel free to reopen if you have any more questions.
